Source/WTF/wtf/text/StringImpl.cpp

```diff
@@ -272,8 +273,12 @@ Ref<StringImpl> StringImpl::create(std::span<const UChar> characters)
     return createInternal(characters);
 }
 
+//static const char* utfStr = "UTF-8";
```

Comment:
There is debugging code left over. This PR is in draft though so maybe that's expected.

Reply:
Yes. I left that in so I could restore it easily. This isn't really ready for review (I'm not sure if it's even a progression yet).

I could use your advise on whether the `TextEncodingRegistry` should accept AtomString as an argument, and bypass the String->StringView->ASCIILiteral->AtomString path.